### PR TITLE
make sure fromUrl uses the the iterator to find the right src

### PR DESCRIPTION
**Pull request by pr1ntr**

### CODE REVIEW

**Comment by pr1ntr** [comment]
Relating to this issue:
https://github.com/GoodBoyDigital/pixi.js/issues/1959

File: src/core/textures/VideoBaseTexture.js [hunk 1]
```diff
@@ -198,7 +198,7 @@ VideoBaseTexture.fromUrl = function (videoSrc, scaleMode)
     {
         for (var i = 0; i < videoSrc.length; ++i)
         {
-            video.appendChild(createSource(videoSrc.src || videoSrc, videoSrc.mime));
+            video.appendChild(createSource(videoSrc[i].src || videoSrc, videoSrc.mime));
```
Comment (englercj) [inline src/core/textures/VideoBaseTexture.js]: I think this should be `videoSrc[i].src || videoSrc[i], videoSrc[i].mime`

Comment (pr1ntr) [inline src/core/textures/VideoBaseTexture.js]: Derp! Resubmitting.

**Comment by pr1ntr** [comment]
Sorry guys, I changed the commit. Big derp on my part.

File: src/core/textures/VideoBaseTexture.js [hunk 2]
```diff
@@ -198,7 +198,7 @@ VideoBaseTexture.fromUrl = function (videoSrc, scaleMode)
     {
         for (var i = 0; i < videoSrc.length; ++i)
         {
-            video.appendChild(createSource(videoSrc.src || videoSrc, videoSrc.mime));
+            video.appendChild(createSource(videoSrc[i].src || videoSrc, videoSrc[i].mime ));
```
Comment (englercj) [inline src/core/textures/VideoBaseTexture.js]: close! Still missing the index on the fallback of the first parameter.

```js
video.appendChild(createSource(videoSrc[i].src || videoSrc, videoSrc[i].mime ));
//should be:
video.appendChild(createSource(videoSrc[i].src || videoSrc[i], videoSrc[i].mime));
`

**Comment by pr1ntr** [comment]
double derp!

**Comment by englercj** [comment]
No! So close! This looks perfect, but the PR is to the master branch and not `dev`!

**Comment by pr1ntr** [comment]
Sorry I am not so handy with the PRs. Do I need to make new one for a different branch?

**Pull request closed without merging**